components/camel-beanstalk/src/main/java/org/apache/camel/component/beanstalk/BeanstalkEndpoint.java

```diff
@@ -42,7 +42,7 @@
     final ConnectionSettings conn;
 
     @UriPath(description = "Connection settings host:port/tube")
-    private String connectionSettings;
+    private final String connectionSettings;
```

Comment:
Is there a specific reason?

Reply:
This needs to be reverted. It's my mistake when i was going through tests.